apps/aehttp/test/aehttp_ws_test_utils.erl

```diff
@@ -60,7 +60,8 @@
                }).
 
 start_link(Host, Port) ->
-    WsAddress = "ws://" ++ Host ++ ":" ++ integer_to_list(Port) ++ "/websocket",
+    % TODO: Make it configurable whether to use `ws` or `wss`
```

Comment:
Is there any use-case in which we want to allow unsecured websockets?

Reply:
I think this should be the default and other options (such as using `wss`) should be documented and made possible at least.